Header header
pcl/PointIndices[] cluster_indices 